packages/desktop/views/account/AccountRouter.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
```

Comment:
We would have to add a file `index.js` in `views/account`, then we could add `AccountRouter` to the import of `'@views'` in `packages/desktop/App.svelte`